resources/prometheus/prometheus-rules.yaml

```diff
@@ -552,14 +565,17 @@ spec:
             max by (namespace, rhacs_instance_id) (
                 central:success_rate10m > bool 0.65
             )
+            or on (namespace, rhacs_instance_id) central:health
             or on (namespace, rhacs_instance_id) central:sli:pod_ready
           record: central:sli:error_rate
 
         # Availability SLI
         # If any of the SLIs is violated, the service counts as unavailable.
         # This is a time series of 0s (down) and 1s (up).
         - expr: |
-            central:sli:pod_ready * on (namespace, rhacs_instance_id) central:sli:error_rate
+            central:sli:pod_ready
+            * on (namespace, rhacs_instance_id) central:health
```

Comment:
What happens here if `probe_success` is undefined? How likely is that?

Reply:
It could be if we misconfigured `ServiceMonitor`, or if blackbox-exporter is down for whatever reason.
I would rate the likelihood of that happening as pretty low...

Reply:
So this means that if blackbox exporter is down, we are blind on other signals like pod ready as well, right? If the probability for this is very low then this may be acceptable, just wanted to point this out.